server/index.js

```diff
@@ -0,0 +1,16 @@
+const dbHelper = require('../database/index.js');
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+
+app.use(bodyParser.json());
+app.use(express.static('../client/public'));
+
+app.get('/reviews/:id', (req, res) => (
+    dbHelper.getListings(req.params.id, function(results) {
+        res.send(results);
+    })
+));
+
+app.listen(3002, () => console.log('Example app listening on port localhost:3002'))
```

Comment:
Do you have a linter? I'm seeing semi-colons missing.

Add http:// to the localhost otherwise I don't think it will open in the browser.

Reply:
I have now set up the Airbnb style guide settings for ESLint